tla-io/src/main/scala/at/forsyte/apalache/io/lir/ItfCounterexampleWriter.scala

```diff
@@ -4,6 +4,7 @@ import at.forsyte.apalache.tla.lir.oper.TlaOper.deinterleave
 import at.forsyte.apalache.tla.lir.oper.{ApalacheOper, TlaFunOper, TlaSetOper}
 import at.forsyte.apalache.tla.lir.values.{TlaBool, TlaInt, TlaStr}
 import at.forsyte.apalache.tla.lir._
+import at.forsyte.apalache.io.lir.NameReplacementMap
```

Comment:
It must have slipped past me at some point in the past, but `NameReplacementMap` is effectively a global `var`. Please get rid of it.